### DISCUSSION TITLE
[辟谣]提防新型诈骗

### DISCUSSION THREAD

**Discussion by MaskedFools-Mochun** [share121/inter-knot#2827] [其他]
介于最近有人冒充法厄同称自己被抓了骗取菲林，我在这声明一下：不要信他，请把给他的菲林打到我账上<img src="https://bbs-static.miyoushe.com/static/2024/01/31/42b5ef8bab5a58fbdd74f17c21c12420_2665774360840406961.png" alt="花火 略略略" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >

**Comment by Nie7bai** [comment]
啊哈<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >
